pkg/kubefed2/federate/util.go

```diff
@@ -49,3 +58,104 @@ func SetBasicMetaFields(resource *unstructured.Unstructured, apiResource metav1.
 		resource.SetNamespace(namespace)
 	}
 }
+
+func getAPIResourceList(config *rest.Config, skipAPIResourceNames string) (map[string]metav1.APIResource, error) {
```

Comment:
As per a previous comment, the name of this function doesn't seem to match what it does.  Consider renaming to something more appropriate (e.g. `apiResourceMap` or `containedAPIResources`).

Reply:
Updated to `namespacedAPIResourceMap`